### DISCUSSION TITLE
Gradient Boosting Survival

### DISCUSSION THREAD

**Discussion by krinetic1234** [sebp/scikit-survival#183] [General]
@sebp Hi I had one last question -- why does Gradient Boosting not offer a `predict_survival_function` method? is `staged_predict`supposed to replace that? I want to generate a function but seem to only be able to do that with Cox models.

**Comment by sebp** [comment]
Gradient boosting supports different losses. If loss="coxph", 
predict_survival_function should probably be made available. In the 
meantime have a look at https://github.com/sebp/scikit-survival/issues/121
